services/QuillLMS/app/models/old_concept_result.rb

```diff
@@ -40,4 +40,27 @@ def correct?
   def concept_uid=(concept_uid)
     self.concept = Concept.where(uid: concept_uid).first
   end
+
+  # Over-riding the `bulk_insert` classmethod from the bulk_insert gem because the
+  # behavior documented in the gem doesn't actually work.  This change makes it behave
+  # as documented when passed `return_primary_keys: true`.  Minimal modifications from
+  # the original code found here, specifically around return values in if blocks:
+  # https://github.com/jamis/bulk_insert/blob/master/lib/bulk_insert.rb
```

Comment:
For future reference, I would add a note here about the open issue:  https://github.com/jamis/bulk_insert/issues/66#issuecomment-938355641

Reply:
Good suggestion.  I'll add a link to the comment.